docs/pages/config-plugins/development-and-debugging.mdx

```diff
@@ -629,7 +629,7 @@ This makes setup a bit easier and helps prevent users from forgetting to add a p
 
 This does come with a couple of caveats:
 
-1. Packages must export a plugin via **app.plugin.js**, this rule was added to prevent popular packages like `lodash` from being mistaken for a config plugin and breaking the prebuild.
+1. Packages can use both use a root **app.plugin.js** file or use the `main` entry point to ship a config plugin. `expo install` only adds config plugins using **app.config.js** automatically to the app manifest. This rule was added added to prevent popular packages like `lodash` from being mistaken for a config plugin and breaking the prebuild.
```

Comment:
```suggestion
1. Packages can use both use a root **app.plugin.js** file or use the `main` entry point to ship a config plugin. `expo install` only adds config plugins using **app.config.js** automatically to the app manifest. This rule was added to prevent popular packages like `lodash` from being mistaken for a config plugin and breaking the prebuild.
```

Reply:
Thanks